lib/templates/dialogue_acts/coref-questions.ts

```diff
@@ -227,6 +227,7 @@ function booleanQuestion(base : Ast.Expression|null, slot : C.FilterSlot) : C.Pa
             schema: slot.schema,
             type: slot.ptype,
             filterable: slot.schema.getArgument(op1.name)!.getImplementationAnnotation<boolean>('filterable') ?? true,
+            symmetric: slot.schema.getArgument(op1.name)!.getImplementationAnnotation<boolean>('symmetric') ?? false,
```

Comment:
What does symmetric mean?

Reply:
for a property `p` in table `t`, `p` is considered "symmetric" if `p` has the same entity type the id of `t`, and if `x` is in `t filter p == y`, then `y` is in `t filter p == x`.

Reply:
also applies to array types, where the second condition would be if `x` is in `t filter contains(p, y)`, then `y` is in `t filter contains(p, x)`.

Reply:
examples: `sister_city`, `siblings`